canonical-web-and-design/juju.is
# Remove the code copyable component with code snippet

## Done
Remove the code copyable component with code snippet

## QA
- Go to the homepage and scroll to the "Getting started" section
- See that the code snippets don't have copy buttons but you can highlight the text and copy it

Fixes https://github.com/canonical-web-and-design/juju.is/issues/9

## Screenshot
![Screenshot_2019-11-06 Juju - The simplest way to deploy and maintain applications in the cloud](https://user-images.githubusercontent.com/1413534/68299896-8aad6280-0094-11ea-9f3d-935f3a1c0060.png)
